python/pyxir/contrib/target/components/DPUCZDX8G/common.py

```diff
@@ -59,3 +59,19 @@ def xgraph_dpu_quantizer(xgraph, inputs_func, **kwargs):
     quantizer = DECENTQuantizer(xgraph, inputs_func, **kwargs)
     q_xgraph = quantizer.quantize()
     return q_xgraph
+
```

Comment:
These two functions shouldn't be here anymore. I removed them to be able to import pyxir on zynq devices without having tensorflow installed. They should now be located in external_quantizer_tools.py